CONTRIBUTING_DOCS.md

```diff
@@ -158,24 +151,51 @@ Here are some other shortcodes:
 - `readfile`: Include the content of another file in the current file, which can be in an arbitrary location.
 - `bootstrap-table`: formats a table using Bootstrap classes; accepts any bootstrap table classes as additional arguments, e.g. `{{< bootstrap-table "table-bordered table-hover" }}`
 
+### Add code to documentation pages
+
+For command, binary and process names, you can use a pair of backticks (\`\<some-name\>\`): `<some-name>`.
+
+Larger blocks of multi-line code text such as configuration files can be wrapped in triple backticks, with a language as a parameter for highlighted formatting.
+
+You can also use the `ghcode` shortcode to embed a single file directly from GitHub:
+
+`{{< ghcode "<https://raw.githubusercontent.com/some-repository-file-link>" >}}`
+
+An example of this can be seen in [content/ngf/get-started.md](https://github.com/nginx/documentation/blob/af8a62b15f86a7b7be7944b7a79f44fd5e526c15/content/ngf/get-started.md?plain=1#L233C1-L233C128), which embeds a YAML file.
+
+### Add images to documentation pages
+
+Use the `img` shortcode to add images to documentation pages. It has the same parameters as the Hugo [figure shortcode](https://gohugo.io/content-management/shortcodes/#figure).
+
+1. Add the image to the `/static/img` directory.
```

Comment:
We include images in other directories as well (e.g. /static/nginx/images, /static/nginx-one/images, /static/him)

- Maybe we should standardize
- But we should talk about it first

Reply:
I think a discussion of standardising the IA for all of our "peripheral" content would be beneficial, since we have already started touch on it for how we organise includes.